v21.2/backup.md

```diff
@@ -19,7 +19,7 @@ You can also backup:
 - [An individual database](#backup-a-database), which includes all of its tables and views.
 - [An individual table](#backup-a-table-or-view), which includes its indexes and views.
 
-    `BACKUP` only backs up entire tables; it _does not_ support backing up subsets of a table.
+    `BACKUP` only backs up entire tables; it **does not** support backing up subsets of a table.
```

Comment:
Could we add something that would indicate what a sub-set of a table could mean -- (e.g. you cannot backup a partition of a regional-by-row table). Although I don't know if "partition" is well-defined. I had to find out from the field team, that it means all rows in the RBR table that are in the same region.